build.sbt

```diff
@@ -8,14 +7,20 @@ import xerial.sbt.Sonatype._
 
 name := "play-html-compressor"
 
-version := "0.4-SNAPSHOT"
+version := "0.4.1"
 
 libraryDependencies ++= Seq(
   "com.googlecode.htmlcompressor" % "htmlcompressor" % "1.5.2",
   "rhino" % "js" % "1.7R2"
 )
 
-lazy val root = (project in file(".")).enablePlugins(PlayScala)
+libraryDependencies += specs2 % Test
+
+libraryDependencies += "org.easytesting" % "fest-assert-core" % "2.0M10"
```

Comment:
Yeah this was changed according to the migration guide: https://www.playframework.com/documentation/2.4.0-RC1/Migration24 
Oddly that paragraph was removed from the 2.4.x link https://www.playframework.com/documentation/2.4.x/Migration24